### PR TITLE
CLOUD-2957 Move AMQ modules out of cct_module

### PR DESCRIPTION
**Pull request by gaohoward**
This commit copies all dependent modules from cct_module repo
  and points modules to local directory.
  after that the build should no longer depends on cct_modules.

Signed-off-by: Howard Gao <howard.gao@gmail.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -76,7 +74,9 @@ modules:
           - name: os-logging
 packages:
       repositories:
-          - jboss-os
+          - name: jboss-os
+            url:
+               repository: http://git.app.eng.bos.redhat.com/git/jboss-container-tools.git/plain/repos/jboss-rhel-os.repo
```
Comment (dsimansk) [inline image.yaml]: @gaohoward beware, that it's internal URL that shouldn't be exposed/leaked in public repository.

**Comment by rcernich** [comment]
You should only be moving the AMQ specific modules (i.e. those modules that only apply to AMQ, not modules that are shared by other images).

File: image.yaml (hunk 1 above)
Comment (gaohoward) [reply image.yaml]: thanks. It's removed.

**Comment by gaohoward** [comment]
@rcernich I've updated the branch. Can you take a look pls? thx.

**Review by rcernich — DISMISSED** [review]
Looks okay to me.  We've been recommending that folks update the module names to prevent duplicates, e.g. amq.jolokia vs os-amq-jolokia.  This will ensure you're picking up the modules from this repo vs. the ones in cct_module.  (Obviously, once the ones in cct_module are deleted, this won't be an issue, but in the meantime...)

**Comment by gaohoward** [comment]
@rcernich thanks Rob. Making different names is a good idea. I'll update my PR soon. Thx again.

**Comment by gaohoward** [comment]
@rcernich done renaming.

**Review by rcernich — APPROVED** [review]
Works for me.  Why did you choose `md-`?

**Comment by gaohoward** [comment]
@rcernich I choose it for no particular reasons. I use it to represent module. :)

**Comment by gaohoward** [comment]
I'll go ahead and merge it.

**Pull request merged**